src/app/components/song-window/song-window.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="title">
+    <a href="{{song.url}}">
+        <h3> {{song.shortTitle}} - {{song.author}} </h3>
+    </a>
+    <span class="custom-br"></span>
+</div>
+<br>
+<div>
+    <iframe width="775" height="436" frameborder="0"
```

Comment:
```suggestion
    <iframe frameborder="0"
```
Il est possible que les valeurs de taille soient inutile ?